### PR TITLE
Implemented contact feature and added FQA section.

### LINKED ISSUE #187: [Feature] Make a working contact form
### Description

We need to enable users to send their queries, via email, to the admin using contact form. I would like to work on this issue under GSSOC'24. I would use emailJS to enable users to send their queries just by submitting the contact form.

### Screenshots

_No response_

### Checklist

- [X] I have checked the existing [issues](https://github.com/pooranjoyb/popShop/issues)
- [X] I have read the [Contributing Guidelines](https://github.com/pooranjoyb/popShop/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)

### PR DESCRIPTION
**Pull request by masterboy376**
Added a FQAs section on contact page and implemented a working contact form on contact page.

Fixes #187

## Type of change

Please give a X on it which is applicable

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactor Code
- [ ] A documentation update
- [ ] Others(mentioned in the issue number)

# How Has This Been Tested?

**Admin have to create an account on EmailJS service, and then create setup a Gmail service and add a template (with variables such as name, email, message). Now, copy EmailJS public key, service id and template id, and then add following fields to the `.env.local` file `VITE_EMAILJS_PUBLIC_KEY`, `VITE_EMAILJS_SERVICE_ID` and `VITE_EMAILJS_TEMPLATE_ID`.
Now simply fill contact form and send.**

# Screenshots and Videos:

[screen-recorder-wed-jun-05-2024-14-47-17.webm](https://github.com/pooranjoyb/popShop/assets/92575005/c10076b7-3e59-461e-bd5c-f7991eb4fcc1)

# Checklist:
give a X on it which is applicable

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by pooranjoyb — COMMENTED** [review]
Great job, @masterboy376! 🎉 Thank you for submitting your pull request. Your contribution is valuable and we appreciate your efforts to improve our project.<p>We will promptly review your changes and offer feedback. Keep up the excellent work! Kindly remember to check our [contributing guidelines](https://github.com/pooranjoyb/popShop/blob/dev/CONTRIBUTING.md)</p>

**Comment by masterboy376** [comment]
@pooranjoyb PTAL

File: src/pages/Contact/FQA.tsx
Comment (pooranjoyb) [inline src/pages/Contact/FQA.tsx:1]: Fix the alignment issues. 
image

**Review by pooranjoyb — CHANGES_REQUESTED** [review]

File: src/pages/Contact/FQA.tsx
Comment (masterboy376) [reply src/pages/Contact/FQA.tsx:1]: fixed it.
[screen-recorder-wed-jun-05-2024-14-47-17.webm](https://github.com/pooranjoyb/popShop/assets/92575005/153acfd2-c60b-4eea-a208-c8a68a09156d)

**Comment by masterboy376** [comment]
Hello @pooranjoyb addressed the comment. PTAL

**Review by pooranjoyb — APPROVED** [review]

**Comment by pooranjoyb** [comment]
Nice work @masterboy376 🚀

**Pull request merged**